service/orchestrator/orchestrator.go

```diff
@@ -82,7 +82,7 @@ type Service struct {
 	// loadMetricsFunc is a function that is used to initially load metrics at the start of the orchestrator
 	loadMetricsFunc func() ([]*assessment.Metric, error)
 
-	catalogsFile string
+	catalogsFiles []string
```

Comment:
I would just have a `catalogsFolder` here and read all the json files that are contained in the folder.